EIPS/eip-1313.md

```diff
@@ -0,0 +1,62 @@
+---
+eip: 1313
+title: Soulmates Token
+description: Bind souls(魂), tokens representing mates.
+author: Jun Kawasaki
```

Comment:
```suggestion
author: Jun Kawasaki (@jun784)
```

Reply:
Thank you for feedback @pandapip1-san, I will update and improve it.